ethminer/DBusInt.h

```diff
@@ -33,7 +33,7 @@ class DBusInt
 				cerr << "Message is null!" << endl;
 			}
 			dbus_message_append_args(msg, DBUS_TYPE_STRING, &hash, DBUS_TYPE_INVALID);
-			if (!dbus_connection_send(conn, msg, NULL)) cerr << "Error sending message!" << endl;
+			if (!dbus_connection_send(conn, msg, nulltr)) cerr << "Error sending message!" << endl;
```

Comment:
There is typo ... nulltr instead of nullptr

Reply:
Yikes. Will fix be but it has been a while since -DETHDBUS has compiled without error.